components/dashboard/src/Menu.tsx

```diff
@@ -255,6 +251,7 @@ export default function Menu() {
             {!isMinimalUI && showTeamsUI && !prebuildId && <div className="flex">
                 {leftMenu.map((entry: Entry) => <TabMenuItem name={entry.title} selected={isSelected(entry, location)} link={entry.link}/>)}
             </div>}
+            {prebuildId && (<div className="flex h-4"></div>)}
```

Comment:
@gtsiolis, please elaborate on https://github.com/gitpod-io/gitpod/pull/4876#discussion_r676629797

This is what I understood from that comment. 

Actually, I dislike the fact that it has to be tweaked in Menu.tsx just for a single page (Prebuild.tsx.)

cc. @jankeromnes, any suggestions?

Reply:
@AlexTugarev Let me add some clarification! 🤓 

The rendering on the right (AFTER) here and in the https://github.com/gitpod-io/gitpod/pull/4876#discussion_r676629797 is what this would ideally look like. 🌟 

Maybe I've miscalculated the spacing when I mentioned the spacing of `4`. 🤷

Reply:
> Actually, I dislike the fact that it has to be tweaked in Menu.tsx just for a single page (Prebuild.tsx.)

I understand how this feels although I like this boring (simple) solution you've added here! Once and if this is needed for more pages we can move this logic to the `Menux.tsx` later. 😃

Reply:
Wait, there is a difference between BEFORE and AFTER? 🤔 🔍

Reply:
Phew! 😅

Reply:
/woof

Reply:
What is even the point of this line? Couldn't it just be removed? I don't see the value of adding an empty div here. 😅

Reply:
> What is even the point of this line? 

Sorry if this is not clear enough @jankeromnes! This line is needed as a) it's used throughout the rest of the pages, b) it adds visual balance for page breadcrumb and header, etc. Let me know if I can add more details why is this line needed. 🙈 

> Couldn't it just be removed? I don't see the value of conditionally adding an empty div here.

From what I understand, because this is the only page currently with no tabs below the team selector, we would have to tweak `Menu.tsx` as @AlexTugarev mentioned above in https://github.com/gitpod-io/gitpod/pull/4970#discussion_r678336197. Do you think that would be a better approach here?

Reply:
> What is even the point of this line?

Sorry, I meant "of this line [of code]". It seems unnecessary to me, and anything it tries to achieve should likely be done differently. I think we should remove it.

@AlexTugarev Happy to have a call about this. 🙂

Reply:
Pasting this only for reference. 🤓 

| With this line of code | Without this line of code |
|-|-|
| <img width="500" alt="line-y" src="https://user-images.githubusercontent.com/120486/127347472-20e481d1-fd3a-4c1f-85e1-d903737aeb2e.png"> | <img width="500" alt="line-n" src="https://user-images.githubusercontent.com/120486/127347476-a0fa9241-4635-4658-b4da-03172477fd15.png"> |

Reply:
![5hvj3e](https://user-images.githubusercontent.com/120486/127347990-8bd6fe47-a7f8-4eb5-90a0-c1180eacfe5f.gif)

Reply:
Awesome! Thanks @jankeromnes for helping out with this! 🎉